.github/workflows/build.yaml

```
@@ -80,6 +80,7 @@ jobs:
         branding: [ RHACS_BRANDING, STACKROX_BRANDING ]
     env:
       ROX_PRODUCT_BRANDING: ${{ matrix.branding }}
+      UI_PKG_INSTALL_EXTRA_ARGS: --ignore-scripts
```

I suggest leaving a small comment here for future-us
```suggestion
      # For consistency with `image/rhel/konflux.Dockerfile`, see comment there.
      UI_PKG_INSTALL_EXTRA_ARGS: --ignore-scripts
```